pkg/internal/request/metric_attributes.go

```diff
@@ -118,3 +120,56 @@ func SpanPeer(span *Span) string {
 
 	return span.Peer
 }
+
+func SpanKindString(span *Span) string {
```

Comment:
I had added `func kindString(span *Span) string` for my JSON PR - coming to think of this now, should we just use that instead of having this here? The returned strings are a bit different, but I believe it's fine to change them if we prefer to return the ones listed in here instead.

Here's the function: https://github.com/grafana/beyla/blob/09c206fdbad572510f9aad79eac1045329e29a08/pkg/internal/request/span.go#L153

Or perhaps even make it `func (s* Span) Kind() string` ? Just an idea, I am sure you have better context than myself to make a call :)

Reply:
We could unify both functions into a single one. The values used here follow the standards of the span metrics generation:
https://grafana.com/docs/tempo/latest/metrics-generator/span_metrics/

The `kindString` values are more readable but I think is worth just avoid duplicating code. I'll rework this as a method.